tests/integration/database-test.js

```diff
@@ -1,5 +1,5 @@
 import {module, test} from 'qunit';
-import Server from 'ember-cli-mirage/server';
+import Server from 'ember-cli-mirage/pretender-server';
```

Comment:
Shouldn't these be able to use `ember-cli-mirage/server` and by not passing `useExpress` they'll default to using Pretender?